### ISSUE TITLE
POD_FIREBASE_VERSION substitution failing?

### ISSUE THREAD

**Issue by jcf120** [chemerisuk/cordova-plugin-firebase-analytics#114]
The generated `PodFile` and `pods.json` contain the raw string "`$POD_FIREBASE_VERSION`" instead of its substituted value, whether that be the default, or one specified manually using the `--variable POD_FIREBASE_VERSION=...`.

E.g.:
```
# DO NOT MODIFY -- auto-generated by Apache Cordova
source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '9.0'

target 'App' do
	project 'App.xcodeproj'
	pod 'Firebase/Analytics', '$POD_FIREBASE_VERSION'
end
```
Is this expected?

Repo with cordova v9.0.0:
```
sudo gem install cocoapods
pod setup
pod repo update

cordova create app com.company.app App
cd app
cordova platform add ios android
cordova plugin add cordova-plugin-firebase-analytics
```
The final command fails with:
```
Failed to install 'cordova-plugin-firebase-analytics': Error: pod: Command failed with exit code 1
```
Running `pod install --verbose` manually gives:
```
ArgumentError - Illformed requirement `"$POD_FIREBASE_VERSION"`
```

**Comment by chemerisuk** [comment]
@jcf120 it's weird, variables work properly in my project.

**Comment by chemerisuk** [comment]
Just double checked, you were right. Fix published in v2.0.3
